package.json

```diff
@@ -40,7 +40,8 @@
     "overrides": {
       "ip@<=2.0.0": ">=2.0.1",
       "minimatch@<3.0.5": ">=3.0.5",
-      "undici@<=5.28.2": ">=5.28.3"
+      "undici@<=5.28.2": ">=5.28.3",
+      "tar@<6.2.1": ">=6.2.1"
```

Comment:
Why was this necessary?

Reply:
because of this: https://github.com/alephium/alephium-frontend/actions/runs/8777788452/job/24083242757